packages/headless/src/features/facets/facet-set/facet-set-slice.ts

```diff
@@ -230,6 +233,29 @@ export const facetSetReducer = createReducer(
         facetRequest.freezeCurrentValues = true;
         facetRequest.preventAutoSelect = true;
       })
+      // TODO: add tests
```

Comment:
This TODO should be removed since you've added tests ?

Reply:
yes. forgot to remove it